Libraries/LibWeb/HTML/Navigator.cpp

```diff
@@ -129,10 +131,30 @@ GC::Ref<CredentialManagement::CredentialsContainer> Navigator::credentials()
 }
 
 // https://w3c.github.io/pointerevents/#dom-navigator-maxtouchpoints
-WebIDL::Long Navigator::max_touch_points()
-{
-    dbgln("FIXME: Unimplemented Navigator.maxTouchPoints");
-    return 0;
+WebIDL::Long Navigator::max_touch_points() const
+{
+    // 1. Let window be this's relevant global object.
```

Comment:
Where does the text for these spec steps come from? They do not match the link above:
https://w3c.github.io/pointerevents/#dom-navigator-maxtouchpoints